website/blog/2023-03-03-databend-weekly-83.md

```diff
@@ -0,0 +1,182 @@
+
+#> SELECT * FROM books;
++------------------------------+---------------------+------+
+| title                        | author              | date |
++------------------------------+---------------------+------+
+| Transaction Processing       | Jim Gray            | 1992 |
+| Readings in Database Systems | Michael Stonebraker | 2004 |
++------------------------------+---------------------+------+
+2 rows in set (0.044 sec)
+```
+
+If you are interested in learning more about how this is achieved, and how it can improve your analytical experience, please refer to these pull requests:
+
+- [PR | feat: backend webhdfs](https://github.com/datafuselabs/databend/pull/10285)
+- [PR | feat: Add support for copying from webhdfs](https://github.com/datafuselabs/databend/pull/10156)
+
+### Support Aggregate Spill to Object Storage
+
+One of the challenges of processing large data volumes is how to perform GroupBy and OrderBy operations efficiently and reliably. To address this challenge, Databend community is working on a new feature that involves spilling intermediate results to cloud-based object storage like Amazon S3.
+
+This feature will enable Databend to handle GroupBy and OrderBy queries with unlimited data size, without running out of memory or compromising performance. If you want to learn more about how this feature works and how it can benefit your analytics needs, please check out this pull request:
+
+- [PR | feat(query): support aggregate spill to object storage](https://github.com/datafuselabs/databend/pull/10273)
+
+### Decimal Data Types
+
+Databend supports various data types for analytics. One of the data types that Databend has recently improved is decimal, which can store exact numeric values with a fixed precision and scale. Decimal types are useful for applications that require high accuracy, such as financial calculations.
+
+```sql
+-- Create a table with decimal data type.
+create table decimal(value decimal(36, 18));
+
+-- Insert two values.
+insert into decimal values(0.152587668674722117), (0.017820781941443176);
+
+select * from decimal;
++----------------------+
+| value                |
++----------------------+
+| 0.152587668674722117 |
+| 0.017820781941443176 |
++----------------------+
+```
+
+If you are interested in learning more about how Databend handles decimal types efficiently and accurately, please read on:
+
+- [Docs | Data Types - Decimal](https://databend.rs/doc/sql-reference/data-types/data-type-decimal-types).
+
+## What's Up Next
+
+We're always open to cutting-edge technologies and innovative ideas. You're more than welcome to join the community and bring them to Databend.
+
+### Initialize `regex` at Compile Time
+
+In `https://github.com/datafuselabs/databend/blob/main/src/query/service/src/servers/federated_helper.rs`, many regular expressions will be used to match queries. We can improve this by initializing [`regex`](https://crates.io/crates/regex) at compile time.
```

Comment:
If you read the code in [federated_helper.rs](https://github.com/datafuselabs/databend/blob/main/src/query/service/src/servers/federated_helper.rs), you may notice that numerous regular expressions are employed to match queries. This can be optimized by initializing the regex at compile time.